### ISSUE TITLE
Mazumder - Explore GISAID and NCBI for XBB.1.5 fastq data

### ISSUE THREAD

**Issue by steph-sing** [FDA-ARGOS/data.argosdb#176]
Requested by Yi and Mazumder

**Comment by Jgergely11** [comment]
I reached out to NCBI about the paired end issue with the XBB.1.5 and was informed that they will be re-uploaded. Still waiting on clarification if they we be re-uploaded as paired-end, or will be designated as single end reads. I will update when I receive further information.

**Comment by steph-sing** [comment]
thanks for the update. This will evolve into a new task in Feb/ as data becomes available.
